src/directive/rateLimit.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  defaultKeyGenerator,
+  rateLimitDirective,
+} from 'graphql-rate-limit-directive';
+import { RateLimiterMemory } from 'rate-limiter-flexible';
+import { GraphQLError } from 'graphql';
+
+export let rateLimiter: CustomRateLimiterMemory;
+
+export class CustomRateLimiterMemory extends RateLimiterMemory {
+  constructor(props) {
+    super(props);
+
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    rateLimiter = this;
```

Comment:
Why are you doing this? Seems too hacky

Reply:
I agree it is a bit hacky. I was not able to access the memory it is natively using to clear out some keys, but with using Redis, we might be able to avoid this.